<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/resources/gear.png</normaloff>:/icons/resources/gear.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tb_PGeneral">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>124</width>
          <height>211</height>
         </rect>
        </property>
        <attribute name="label">
         <string>General</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tb_PHotkeys">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>124</width>
          <height>211</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Hotkeys</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="PGeneral">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="margin">
          <number>6</number>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="verticalSpacing">
            <number>9</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Database file</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="le_DatabaseFile"/>
             </item>
             <item>
              <widget class="QToolButton" name="tb_SelectDatabaseFile">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/resources/folder-open.png</normaloff>:/icons/resources/folder-open.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Language</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cb_Language">
             <item>
              <property name="text">
               <string>English</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Russian</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Change font size step</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sb_ChangeFontSizeStep">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Default color theme</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="cb_DefaultColorTheme">
             <item>
              <property name="text">
               <string>Red</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Blue</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Orange</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>New note text template</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="le_NewNoteTextTemplate">
               <property name="toolTip">
                <string>Template wich used for new notes. You may, for example, set your sign in template.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tb_SelectNewNoteTextTemplate">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/resources/folder-open.png</normaloff>:/icons/resources/folder-open.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Note showing template</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLineEdit" name="le_NoteShowingTemplate">
               <property name="toolTip">
                <string>Template for the showing notes. It has special placeholders for note data: title, text, datetime, colors.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tb_SelectNoteShowingTemplate">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/resources/folder-open.png</normaloff>:/icons/resources/folder-open.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QCheckBox" name="cb_RunOnSystemStartup">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Run as system startup</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="l_WelcomeString">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;This is first run of the program.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Please, set your preferences and press OK.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="p_Hotkeys">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Create note:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_2">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Notes window:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_3">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Close application:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="lineEdit_4"/>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_4">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBox</sender>
   <signal>currentChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tb_SelectDatabaseFile</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>selectDatabaseFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tb_SelectNewNoteTextTemplate</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>selectNewNoteTextTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tb_SelectNoteShowingTemplate</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>selectNoteShowingTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectDatabaseFile()</slot>
  <slot>selectNewNoteTextTemplate()</slot>
  <slot>selectNoteShowingTemplate()</slot>
 </slots>
</ui>
